### DISCUSSION TITLE
how to paste to file picker from * register ?

### DISCUSSION THREAD

**Discussion by noor-tg** [helix-editor/helix#11722] [Q&A] [answered]
I copied text with `*` while selected. I want to search for this text as file name with filepicker.
How to do it ?

**Comment by noor-tg** [comment]
for now at least I need to copy to system clipboard with `<space>y` then `<ctrl><shift>v`.

**Comment by the-mikedavis** [comment, marked as answer]
You can use `<C-r>/`. `*` copies the selection contents to the `/` register and you can use `<C-r>` followed by a register name to paste the contents into any prompt. This works for prompts like command mode (`:`) and also pickers since the input in the upper-left corner is implemented as a prompt.

**Reply by noor-tg** [reply]
thank you
